atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -197,6 +201,10 @@ private static SerializableTransactionManager create(
                 SimpleTimeDuration.of(config.getDefaultLockTimeoutSeconds(), TimeUnit.SECONDS));
         LockAndTimestampServices lockAndTimestampServices = createLockAndTimestampServices(
                 config,
+                JavaSuppliers.compose(runtimeConfig ->
+                                runtimeConfig.map(AtlasDbRuntimeConfig::timestampClient)
+                                        .orElse(ImmutableTimestampClientConfig.builder().build()),
```

Comment:
This looks like it will potentially create a new config on every timestamp call. Probably not a huge deal but might want to make this a constant